natgeo/mortar
# AEM-6920: Finalize Broadsheet Styling - Text

https://jira.natgeo.com/browse/AEM-6920

To test:

- Pull AEM-6920-rebased
- `npm test`
- `gulp serve`
- Verify broadsheet layout changes at various breakpoints
